src/context/ContextType.ts

```diff
@@ -6,13 +6,15 @@ export enum ContextTypes {
   Country = 'fdc3.country',
   Currency = 'fdc3.currency',
   Email = 'fdc3.email',
+  Interaction = 'fdc3.interaction',
   Instrument = 'fdc3.instrument',
   InstrumentList = 'fdc3.instrumentList',
```

Comment:
Alphabetical order
```suggestion
  "Instrument": ["https://fdc3.finos.org/schemas/next/instrument.schema.json"],
  "InstrumentList": ["https://fdc3.finos.org/schemas/next/instrumentList.schema.json"],
  "Interaction": ["https://fdc3.finos.org/schemas/next/interaction.schema.json"],
```

also added to #747